.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Do we wanna keep this in the repo? I would prefer not to, but instead update the `.gitignore` to automatically ignore "personal configs".